src/components/Summary.jsx

```diff
@@ -20,69 +20,72 @@ const Summary = () => {
 
   const urls = useMemo(() => {
     const base = {
-      temperature_unit: preferences.temperature_unit.symbol,
-      conductivity_unit: preferences.conductivity_unit.symbol,
-      include_measurements: false,
+      temperatureUnit: preferences.temperatureUnit.symbol,
+      conductivityUnit: preferences.conductivityUnit.symbol,
+      includeMeasurements: false,
     };
     return {
       current: urlWithParams(ENDPOINT, {
         ...base,
-        start_date: formatISO(startDate),
-        end_date: formatISO(endDate),
-        location_name: preferences.location.symbol,
+        startTime: formatISO(startDate),
+        endTime: formatISO(endDate),
```

Comment:
perhaps change to
`endTime: formatISO(endOfDay(endDate)),`?
to include the entire date, currently measurements from endDate won't be included since it will use `endTime: "<endDate>T00:00:00Z"`